README.md

```
@@ -91,19 +1038,299 @@ Benefits:
+
+| Parameter | Required | Description |
+|-----------|----------|-------------|
+| `host` | Yes | Extractor host name (e.g., `TurboVidPlay`, `Vidoza`) |
+| `d` | Yes | Destination URL (the video page URL to extract) |
+| `api_password` | Yes* | API password (*if configured) |
+| `redirect_stream` | No | If `true`, returns 302 redirect to the proxied stream URL |
+
+#### Example Usage
+
+**Get extraction result as JSON:**
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)